examples/p5js/NeuralNetwork/NeuroEvolution_basics/sketch.js

```diff
@@ -28,10 +28,11 @@ function setup() {
   const options = {
     inputs: 3,
     outputs: ['yes', 'no'],
-    task: 'classification'
+    task: 'classification',
+    static: true
```

Comment:
@shiffman - this `static` property is read into the constructor

Reply:
Oh, I love this, thank you! Great solution to have an extra flag in the options. Why did you pick the term "static"? I don't have a better suggestion necessarily, but I'm not sure that describes the idea of "no training data" exactly?

Reply:
oh! Not sure exactly. I kind of thought of it as the difference btween a static "brain" vs a dynamic one 😹 I'm ok to change it to anything else!